packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/languages/perl/changeList.yml

```diff
@@ -0,0 +1,22 @@
+languageId: perl
+command:
+  version: 4
+  spokenForm: change list
+  action: { name: clearAndSetSelection }
+  targets:
+    - type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: { type: list }
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: '@array = ( "one", 2, "_tres" )'
```

Comment:
Worth adding "item" tests for these as well. If you haven't implemented `collectionItem`, then you'll just get these from the default impl

Reply:
"item" tests added